sdk/tests/conformance2/reading/format-r11f-g11f-b10f.html

```diff
@@ -95,25 +114,34 @@
   clearAndVerifyColor(width, height, true, false, false, 1000);
   clearAndVerifyColor(width, height, true, false, false, 2500);
   clearAndVerifyColor(width, height, true, false, false, 5000);
-  clearAndVerifyColor(width, height, true, false, false, 8191);
-  clearAndVerifyColor(width, height, true, false, false, 8192);
```

Comment:
Just curious: why do you remove 8192?  As that value leads to Infinity on Mac rather than NaN, so slightly different.